ibmcloud/image/build.sh

```diff
@@ -113,7 +113,8 @@ fi
 
 printf "\nCopying partitions from the base image $src_img_path\n"
 
-qemu-img create -f qcow2 -b "$base_img_path" "$src_img_path" $disksize
+cp -f "$base_img_path" "$src_img_path"
```

Comment:
Is https://github.com/confidential-containers/cloud-api-adaptor/blob/main/podvm/Makefile#L76 need be updated too?

Reply:
@liudalibj the problem is `-b "$base_img_path"`, I think it's OK for code in Makefile.